kotori/daq/graphing/grafana/dashboard.py

```diff
@@ -302,8 +307,9 @@ def use_field(field_name: str):
 
 class GrafanaDashboard(object):
 
-    def __init__(self, channel=None, uid=None, title='default', datasource='default', folder_id=None, dashboard_data=None):
+    def __init__(self, channel=None, model=None, uid=None, title='default', datasource='default', folder_id=None, dashboard_data=None):
         self.channel = channel or Munch()
+        self.model: GrafanaDashboardModel = model
         self.dashboard_uid = uid
         self.dashboard_title = title
```

Comment:
When propagating the whole `model` instance to those functions, it looks like we could stop propagating `dashboard_uid`, `dashboard_title`, and maybe more, in order to trim the interface/signature.